utils/coreutils/patches/001-no_docs_man_tests.patch

```diff
@@ -7,29 +9,20 @@
 -SUBDIRS = po . gnulib-tests
 +SUBDIRS = po
  
- changelog_etc =				\
-   ChangeLog-2005			\
-@@ -213,6 +213,4 @@ AM_CPPFLAGS = -Ilib -I$(top_srcdir)/lib
+ EXTRA_DIST =				\
+   .mailmap				\
+@@ -210,6 +210,3 @@ AM_CPPFLAGS = -Ilib -I$(top_srcdir)/lib -Isrc -I$(top_srcdir)/src
  
  include $(top_srcdir)/lib/local.mk
  include $(top_srcdir)/src/local.mk
 -include $(top_srcdir)/doc/local.mk
```

Comment:
They pull in automake magic for docs and tests which we want to avoid while building. Usually its just a matter of patching the sole `SUBDIRS:=...` line but in this case it seems a lot of "cleverness" went into the automake file.